src/merge_operator.rs

```diff
@@ -41,7 +41,8 @@
 //!let mut opts = Options::default();
 //!
 //!opts.create_if_missing(true);
-//!opts.set_merge_operator("test operator", concat_merge, None);
+//!opts.set_merge_operator_associative("test operator", concat_merge);
+//!//opts.set_merge_operator("test operator", concat_merge, partial_concat_merge); // if your merge is not associative
```

Comment:
I suppose this comment could be deleted as well ?

Reply:
Oops…
done !

Reply:
Thanks.